packages/headless/src/features/result-preview/result-preview-insight-analytics-actions.ts

```diff
@@ -0,0 +1,34 @@
+import {InsightPanel} from '@coveo/relay-event-types';
+import {Result} from '../../api/search/search/result';
+import {
+  ClickAction,
+  analyticsEventItemMetadata,
+  documentIdentifier,
+  makeInsightAnalyticsActionFactory,
+  partialDocumentInformation,
+  validateResultPayload,
+} from '../analytics/analytics-utils';
+import {analyticsEventCaseContext} from '../analytics/insight-analytics-utils';
+
+export const logDocumentQuickview = (result: Result): ClickAction => {
```

Comment:
I don't understand?
What was it before this PR? Just a regular ItemClick with Quickview as the "cause" ?

Reply:
Indeed, before this PR it was a regular itemClick exactly like we send for the search use case.